test/unit/presenters/publishing_api/topical_event_presenter_test.rb

```diff
@@ -41,6 +41,7 @@ class PublishingApi::TopicalEventPresenterTest < ActiveSupport::TestCase
       details: {
         about_page_link_text: topical_event.about_page.read_more_link_text,
         body: Whitehall::GovspeakRenderer.new.govspeak_to_html(topical_event.description),
+        emphasised_organisations: topical_event.lead_organisations.map(&:content_id),
```

Comment:
This topical event actually has any lead organisations set, this test also passes if you replace `topical_event.lead_organisations.map(&:content_id)` with `[]`. It would be nice to test it with some actual data. 

I'm also not a fan of creating the test assertion with the same code as the implementation when you can avoid it as (I know it's done in the above two lines too but maybe harder with govspeak). If there's something off with your implementation you wouldn't spot it - so I'd prefer just doing the dumb thing of putting the array of content ids in here (you could create the lead organisations from a variable set to the array of content ids, and reuse that here, so it's clear what the link is).

Reply:
have sorted this, going to merge!